setup.sh

```diff
@@ -84,12 +97,13 @@ action() {
             # Install conda if necessary
             if [ ! -f "miniconda/bin/activate" ]; then
                 # Miniconda version used for all environments
-                MINICONDA_VERSION="Miniconda3-py39_4.10.3-Linux-x86_64"
+                # MINICONDA_VERSION="Miniconda3-py39_4.10.3-Linux-x86_64"
+                MINICONDA_VERSION="Miniconda3-latest-Linux-x86_64"
```

Comment:
I would be in favor of pinning this version, instead of using latest, to avoid breaking when setting up a new env

Reply:
I am setting up the updated env with `23.5.2` so we should pin that version using `Miniconda3-py39_23.5.2-0-Linux-x86_64`